src/index.ts

```diff
@@ -6,3 +6,5 @@ export type {
   LogicalOperatorNewline,
   FormatOptions,
 } from './FormatOptions';
+export { default as Formatter } from './formatter/Formatter';
+export { default as Tokenizer } from './lexer/Tokenizer';
```

Comment:
Thank you so much for this quick PR! Could `{ expandPhrases }` also be exported, please? I find it to be very useful.

Thanks again!

Reply:
Good point. Added it.